Cargo.toml

```diff
@@ -32,3 +32,7 @@ mio = { git = "https://github.com/servo/mio.git", branch = "servo" }
 winapi = { git = "https://github.com/servo/winapi-rs", branch = "patch-1" }
 spirv_cross = { git = "https://github.com/servo/spirv_cross", branch = "wgpu-servo" }
 backtrace = { git = "https://github.com/MeFisto94/backtrace-rs", branch = "fix-strtab-freeing-crash" }
+
+[patch."https://github.com/servo/webxr"]
+webxr = { git = "https://github.com/jdm/webxr", branch = "surfman-0.1" }
+webxr-api = { git = "https://github.com/jdm/webxr", branch = "surfman-0.1" }
```

Comment:
I had to use jdm instead of servo here because cargo requires a different source for a patch, and a branch from the same repository does not count.

Reply:
Why not just `cargo update -p webxr --precise <hash>` instead? we can later mark the hash with a tag.

Reply:
Smart.